plugins/rect/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = (rem) => ({ addUtilities }) => {
+  const rect = {
+    '.rect-target': {
+      width: rem(44),
+      height: rem(44)
```

Comment:
Being that this plugin is in the `require()`-able plugin directory, we'll need to rewrite it to support config options. I'm thinking just looping over an array of numbers to generate the utilities.